app/api/contributors/index.js

```diff
@@ -46,6 +46,13 @@ router.get('/', restricted, async (req, res) => {
   `, { requestedSkills: skills });
 
   const users = (await query).rows;
+  
+  var i;
+  for(i = 0; i < users.length; i++){
```

Comment:
**nit**: There's no need to define `i` before the loop, and we prefer `let`/`const` to `var`. Try: `for (let i = 0; i < users.length; i++) {`